packages/apps/docs/src/components/Search/Search.tsx

```diff
@@ -35,6 +36,14 @@ export const Search: FC<IProps> = ({ query, hasScroll, limitResults }) => {
       if (tabName === 'docs') {
         handleSemanticSubmit(query);
       }
+
+      analyticsEvent(
+        EVENT_NAMES[tabName === 'qa' ? 'search:qa' : 'search:docs'],
+        {
+          label: query,
```

Comment:
```suggestion
          query: query,
```
i think `query` makes it more sense instead of label.

Reply:
Unfortunately these values are the parameter names in the method `analyticsEvent`, not sure it makes sense to change it there:
https://github.com/kadena-community/kadena.js/blob/b100166cb29fa26d5d194d17be0321260ea23c16/packages/apps/docs/src/utils/analytics.ts#L21